tests/integration/docker/test_docker.py

```diff
@@ -722,6 +722,18 @@ def test_run_with_additional_arguments(self, docker_client: ContainerClient):
         assert env_variable in stdout
         assert "EXISTING_VAR=test_var" in stdout
 
+    def test_run_with_additional_arguments_add_host(self, docker_client: ContainerClient):
+        additional_flags = "--add-host sometest.localstack.cloud:127.0.0.1"
+        stdout, _ = docker_client.run_container(
+            "alpine",
+            remove=True,
+            command=["getent", "hosts", "sometest.localstack.cloud"],
+            additional_flags=additional_flags,
+        )
+        stdout = stdout.decode(config.DEFAULT_ENCODING)
```

Comment:
nit: We could use the `to_str(..)` util here. (let's not bother changing for this PR.. 👍 )